src/include/miopen/hipoc_kernel.hpp

```diff
@@ -145,12 +145,12 @@ struct HIPOCKernelInvoke
         memcpy(hip_args, &(any_args[0].buffer[0]), any_args[0].size());
         //        copy_arg(any_args[0], hip_args, 0);
 
-        for(unsigned long idx = 1; idx < any_args.size(); idx++)
+        for(size_t idx = 1; idx < any_args.size(); idx++)
```

Comment:
[R] I am not sure if plain `size_t` is portable for C++. I think it is better to use `std::size_t` (at least where you insert it in this PR). Up to you.

Reply:
Taken from https://open-std.org/jtc1/sc22/wg21/docs/wp/html/nov97-2/lib-intro.html
> 17.4.3.1.4  Types                                   [lib.extern.types]
> 1 For  each  type  T  from  the Standard C library,26) the types ::T and
>  std::T are reserved to the implementation and, when defined, ::T shall
>  be identical to std::T.

So it's not a problem to use size_t, but it's best to completely avoid including or at least using stuff defined c headers when it's possible.